jina/helper.py

```diff
@@ -693,12 +693,16 @@ def run(self):
         return asyncio.run(func(*args, **kwargs))
 
 
-def check_keys_exist(keys_to_check, existing_keys):
-    missed = []
-    for k in keys_to_check:
-        if k not in existing_keys:
-            missed.append(k)
-    return missed
+def check_keys_exist(keys: Iterator, keys_existing: Iterator, save_abspath: str, logger: 'JinaLogger'):
```

Comment:
the name save abspath of this argument links it heavily with an executor, I would call it check_path

Reply:
Done